plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/testing/TpchHudiTablesInitializer.java

```diff
@@ -200,7 +200,7 @@ private HoodieJavaWriteClient<HoodieAvroPayload> createWriteClient(TpchTable<?>
         }
 
         HoodieIndexConfig indexConfig = HoodieIndexConfig.newBuilder().withIndexType(HoodieIndex.IndexType.INMEMORY).build();
-        HoodieCompactionConfig compactionConfig = HoodieCompactionConfig.newBuilder().archiveCommitsWith(20, 30).build();
+        HoodieCompactionConfig compactionConfig = HoodieCompactionConfig.newBuilder().build();
```

Comment:
Can you build archival configs using `HoodieArchivalConfig` and set it to write config. I have a [commit](https://github.com/codope/trino/commit/d4e5d54f44afc316b0c2ccfb1d66aaaf08355298) upgrading hudi version to 0.12.1-rc1. You can take a look for reference.

Reply:
Thanks @codope , I am working with my colleage on MOR, there is reflect issue when we use 0.11.1, so i upgraded to 0.12.1 to resolve issue on JDK 17. Next time, I will talk with you first.